pkg/api/types.go

```diff
@@ -2035,36 +2053,35 @@ type ComponentStatusList struct {
 // will be populated based on the Container configuration if they are not set.  Defining them on
 // both the Container AND the SecurityContext will result in an error.
 type SecurityContext struct {
-	// Capabilities are the capabilities to add/drop when running the container
+	// Capabilities are the capabilities to add/drop when running containers.
+	// Optional: Defaults to the default set of capabilities granted by the container
```

Comment:
I can be free on Monday.

On Sat, Oct 17, 2015 at 1:58 PM, Paul Morie <notifications@github.com>
wrote:

> In pkg/api/types.go
> <https://github.com/kubernetes/kubernetes/pull/14705#discussion_r42312078>
> :
>
> > @@ -2035,36 +2053,35 @@ type ComponentStatusList struct {
> >  // will be populated based on the Container configuration if they are not set.  Defining them on
> >  // both the Container AND the SecurityContext will result in an error.
> >  type SecurityContext struct {
> > -	// Capabilities are the capabilities to add/drop when running the container
> > +	// Capabilities are the capabilities to add/drop when running containers.
> > +	// Optional: Defaults to the default set of capabilities granted by the container
>
> Going to hold off changing this until we recertify that we have agreement
> on what the semantics should be.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/14705/files#r42312078>.
>